deltachat-ios/DC/DcContext.swift

```diff
@@ -102,9 +102,9 @@ public class DcContext {
         return Int(dc_send_videochat_invitation(contextPointer, UInt32(chatId)))
     }
 
-    public func getChatMsgs(chatId: Int) -> [Int] {
+    public func getChatMsgs(chatId: Int, flags: Int32) -> [Int] {
```

Comment:
```suggestion
    public func getChatMsgs(chatId: Int, includingDayMarker: Bool = true) -> [Int] {
```
or 
```suggestion
    public func getChatMsgs(chatId: Int, options: GetChatMsgOptions = .includeDayMarker) -> [Int] {
```
I think the Swift API should be easily understandable from call site

Reply:
using a boolean seems fine. 

for duplicating "everything" to a swift enum or sth - not sure, at least we did not do that in the past. unless generated automatically, that's a error-prone and/or need quite some maintenance - we're doing that for Java, as we have to - and are quite happy that you can use C from Swift as a first-class thing. in fact, iirc, not so far away we fixed bugs as the swift-enum differ from the C-ones ...

so, if in doubt, i can happily life with some C-constants - esp. as the writer of the glue code is the same as the caller of the glue code for the foreseeable future :)

also, the more far we go away from the C-bindings, the more there is a need for a dedicated documentation - up to now, we can just work with https://c.delta.chat

Reply:
Sure, I personally found the `flags: 0` not very self explaining which is why I commented, but you're the boss.